runhouse/servers/cluster_servlet.py

```diff
@@ -377,6 +377,59 @@ async def _status_for_env_servlet(self, env_servlet_name):
         except Exception as e:
             return {"env_servlet_name": env_servlet_name, "Exception": e}
 
+    def _get_cpu_usage(self, system_cpu_utilization: Iterable):
+        """
+        # TODO [SB]: add documentation
+        :param system_cpu_utilization:
+        :return:
+        """
+        cpus = []
+        percent = []
+        current_val = next(system_cpu_utilization, None)
```

Comment:
we should be iterating over this object, not just taking the first one, to get more precise measurements